.travis.yml

```diff
@@ -64,14 +65,37 @@ jobs:
         token:
           secure: dDlkIawHcODlW9B/20/cQCtzeoocvs0hKuNngRKXKqzXLWTRq33oq/B7+39tAixWbmv6exTpijiKrRNFiSCW5Z4iwHLwaRD4XJznxw63e/Hus/dxg2Tvqx7XFpkCz8mT1Z+gZQE5YxAngeZPpI/sZbZtF1UO3yH5eLeeokZ15p26ZskQUPoYuzrTgTzYL3XfpG3F+20rNBawH1ycsCTVD/08/n31d2m3CrKAsbW7er92ek6w4fzKr7NW8WeXjrPJETVpw5fQg1Od3pRGW8dPQaJcvKQEogMp8Mm0ETYd0qigg89/giBz7QwOgmAWQ4dH+DfZH4Ojl//127QztBolMvyDMQBykWrtJoGcij05sT6K2IJr2FHeUBO12MAEdjiVvhQj3DtTzjPiZAHHDBSLWxLKWWhlhHE4pq7g1MQhqXkaAHI2BLNzwLmaowbMT0bECf9yfz6xx18h6XPQFX44oOktraobVALFlyHqeKa8zdcUt22LF6uAL1m5dxL0tny3eXCIPE4UH/RZgua/cHV9G3cUvKQa/QnFSLRhvWVSbGB+7YsHouBJcsUOOW1gmd5442XuC7mpppccRldh+GSxUk6TBJRAx7TeQ0ybDUaoco9MUqp2twv3KreR2+8Q12PDaAhfQVNEGdF3wTm1sShImjCN4VN3eSLlBEbve1QRQXM=
         skip_cleanup: true
-        draft: true
         prerelease: true
         overwrite: true
+        edge: true
         name: ${TRAVIS_TAG:-edge}
-        body: $TRAVIS_COMMIT_MESSAGE
+        release_notes: $TRAVIS_COMMIT_MESSAGE
         file: build/bin/RelWithDebInfo/solvespace.exe
         on:
           repo: solvespace/solvespace
+    - stage: deploy
+      name: "Windows with OpenMP"
+      os: windows
+      install: .travis/install-windows.sh
+      script: .travis/build-windows.sh release openmp
+      before_deploy:
```

Comment:
Should we de-duplicate this before_deploy script and put it in a separate file? Or as a YAML anchor?

Reply:
I think a bit of duplication is not a big deal, I for one have trouble understanding the YAML anchor from Snap.